test/e2e/virtual/initializingworkspaces/virtualworkspace_test.go

```diff
@@ -581,12 +580,5 @@ func workspaceLabelsUpToDate(t *testing.T, workspace tenancyv1alpha1.ClusterWork
 		t.Logf("workspace %s phase label is not updated yet", workspace.Name)
 		return false
 	}
-	for _, initializer := range workspace.Status.Initializers {
-		key, value := initialization.InitializerToLabel(initializer)
-		if got, exists := workspace.ObjectMeta.Labels[key]; !exists || got != value {
-			t.Logf("workspace %s initializer labels are not updated yet", workspace.Name)
-			return false
-		}
-	}
```

Comment:
hmm, not good to remove

Reply:
okay, that means we need to make sure the labels are attached to an objt.